Tests/StreamChatTests/APIClient/Endpoints/EndpointPath_Tests.swift

```diff
@@ -6,46 +6,51 @@
 import XCTest
 
 final class EndpointPathTests: XCTestCase {
-    func sendMessage_shouldBeQueuedOffline() throws {
-        try XCTAssertTrue(EndpointPath.sendMessage(ChannelId(cid: "")).shouldBeQueuedOffline)
+    func test_sendMessage_shouldBeQueuedOffline() throws {
+        XCTAssertTrue(EndpointPath.sendMessage(.unique).shouldBeQueuedOffline)
     }
 
-    func editMessage_shouldBeQueuedOffline() {
+    func test_editMessage_shouldBeQueuedOffline() {
         XCTAssertTrue(EndpointPath.editMessage("").shouldBeQueuedOffline)
     }
 
-    func deleteMessage_shouldBeQueuedOffline() {
```

Comment:
These tests were not even running, because they did not have the `test_` prefix 😅